### DISCUSSION TITLE
[Question] Label의 스타일을 prop으로 덮을 수 없을까요?

### DISCUSSION THREAD

**Discussion by jy-son** [wanteddev/montage-web#59] [Q&A]
### 요약

Checkbox, Label에 대한 가이드대로 사용하였을 때, Label의 variant, weight, color를 오버라이드 할 수 없습니다.

### 상세

```
<FlexBox>
  <Checkbox />
  <Label variant="heading1">label</Label> // <- Not working !
</FlexBox>
```

<img width="617" alt="image" src="https://github.com/wanteddev/wds/assets/89887036/eff42b07-1002-4629-9bc5-3aa29af1bacb">

<img width="712" alt="image" src="https://github.com/wanteddev/wds/assets/89887036/9f44cf7a-69e5-4438-99d0-ad7badc86e45">

**Comment by jy-son** [comment]
정상동작으로 닫습니다!

**Reply by Sh031224** [reply]
정리

Label을 단독으로 사용할 때는 문제가 없으나 Radio, Checkbox 와 같이 컨트롤 요소의 형제 요소로 위치하는 경우
Label에서 특정 스타일로 고정을 하기 때문에 props로 주어도 동작을 따로 하지 않는다.

```css
checkbox ~ label {
 // style
}
```

그렇기에 props가 아닌 스타일 오버라이드로 사용해야함
